pom.xml

```diff
@@ -306,6 +316,51 @@
 			<version>${debezium.version}</version>
```

Comment:
Do we still need the Debezium dependency if you replaced it with the `EmbeddedKafkaCluster`?